README.md

```diff
@@ -116,13 +116,15 @@ PBS_API_TOKEN_NAME='user@pam!prometheus'
 PBS_API_TOKEN='123e4567-e89b-12d3-a456-426614174000'
 PBS_URL='https://pbs.example.com'
 PUSHGATEWAY_URL='https://pushgateway.example.com'
+PBS_TASK_PERIOD_DAYS=24
 ```
 
 - `PBS_API_TOKEN_NAME` should be the value in the "Token name" column in the Proxmox Backup Server user interface's `Configuration - Access Control - Api Token` page.
 - `PBS_API_TOKEN` should be the value shown when the API Token was created.
   - This token should have at least the `Datastore.Audit` access role assigned to it and the path set to `/datastore`.
 - `PBS_URL` should be the same URL as used to access the Proxmox Backup Server user interface
 - `PUSHGATEWAY_URL` should be a valid URL for the [push gateway](https://github.com/prometheus/pushgateway).
+- `PBS_TASK_PERIOD_DAYS` can be used to overwrite default of 24 hours for the tasks reporting period.
```

Comment:
@ebuzzz shouldn't it be `PBS_TASK_PERIOD_HOURS` instead? Or you could change the date variable to days if you prefer: 
`twenty_four_hours_ago=$(date +%s --date "${PBS_TASK_PERIOD_DAYS} days ago")`

Reply:
notice `... days ago` instead of `... hours ago`

Reply:
notice `... days ago`  instead of `... hours ago`